Ilya/NetflixRoulette/netflix-roulette/src/shared/movieCard/MovieCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import "./MovieCard.scss";
+import film from "../../images/film.jpg";
+
+class MovieCard extends React.Component {
+    state = {
```

Comment:
no need in keeping film inside of the state. You don't change it. You need to pass film as prop